src/components/sbb-image/sbb-image.tsx

```diff
@@ -385,12 +353,76 @@ export class SbbImage {
       }`;
     }
 
-    const configs = pictureSizesConfigData(pictureSizesConfig);
+    return imageHelperGetBreakpoints(pictureSizesConfig);
+  }
+
+  private _createMediaQueryString(mediaQueries): string {
+    let mediaQuery = '';
+
+    mediaQueries.forEach((mq) => {
+      const mqCondition = mq.conditionFeature;
+      let mqValue;
+
+      if (mq.conditionFeatureValue.lyneDesignToken) {
+        mqValue = this._matchMediaQueryDesignToken(mq.conditionFeatureValue.value);
+      } else {
+        mqValue = mq.conditionFeatureValue.value;
+      }
+
+      const mqCombiner = mq.conditionOperator ? ` ${mq.conditionOperator} ` : '';
+
+      mediaQuery += `(${mqCondition}: ${mqValue})${mqCombiner}`;
+    });
+
+    return mediaQuery;
+  }
+
+  public render(): JSX.Element {
+    let { caption } = this;
+    let schemaData = '';
+    const variantClass = this.variantTeaserHero ? ` image__figure--teaser-hero` : '';
+    let additionalClasses = '';
+
+    if (this.noRadius) {
+      additionalClasses = ` image__figure--no-radius`;
+    }
+
+    if (this.aspectRatio) {
+      additionalClasses += ` image__figure--ratio-${this.aspectRatio}`;
+    }
+
+    const attributes: {
+      role?: string;
+    } = {};
+
+    const imageUrlLQIP = this._prepareImageUrl(this.imageSrc, true);
+    const imageUrlWithParams = this._prepareImageUrl(this.imageSrc, false);
+
+    if (this.loading === 'lazy') {
+      this.decoding = 'async';
+      this.importance = 'low';
+    }
+
+    if (this.copyright) {
+      caption = `${this.caption} ©${this.copyright}`;
+      schemaData = `{
+        "@context": "https://schema.org",
+        "@type": "Photograph",
+        "image": "${this.imageSrc}",
+        "copyrightHolder": {
+          "@type": "${this.copyrightHolder}",
+          "name": "${this.copyright}"
+        }
+      }`;
+    }
 
-    const variantClass = this.variant ? ` image__figure--${this.variant}` : '';
+    const pictureSizeConfigs = this._preparePictureSizeConfigs();
 
     return (
-      <figure class={`image__figure${variantClass}${this._loadedClass}`} {...attributes}>
+      <figure
+        class={`image__figure${variantClass}${this._loadedClass}${additionalClasses}`}
```

Comment:
because there are a lot of classes and string concatenation is done manually at the moment, I suggest to use the following notation (copied from divider).
```
        class={{
          'sbb-divider': true,
          [`sbb-divider--${this.orientation}`]: true,
          'sbb-divider--negative': this.negative,
        }}
```